### PR TITLE
TICtxtWrd-n:Test Improvements, Context Wording, for the ecosystem npm and yarn.

### PR DESCRIPTION
**Pull request by thavaahariharangit**
Test Improvements, Context Wording, for the ecosystem npm and yarn.

### CODE REVIEW

File: npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb [hunk 1]
```diff
@@ -93,7 +93,7 @@
     end
   end
 
-  context "that has a blank file: in the package-lock" do
+  context "when it has a blank file: in the package-lock" do
```
Comment (abdulapopoola) [inline npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb:96]: what's "it"?

File: npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb [hunk 2]
```diff
@@ -93,7 +93,7 @@
     end
   end
 
-  context "that has a blank file: in the package-lock" do
+  context "when repo has a blank file: in the package-lock" do
```
Comment (jurre) [inline npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb:96]:
```suggestion
  context "when the repo has a blank file: in the package-lock" do
```

File: npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb [hunk 3]
```diff
@@ -200,7 +200,7 @@
           )
       end
 
-      context "that has an unfetchable path" do
+      context "when path is unfetchable" do
```
Comment (abdulapopoola) [inline npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb:203]:
```suggestion
      context "when the path is unfetchable" do
```

File: npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npmrc_builder_spec.rb [hunk 4]
```diff
@@ -48,7 +48,7 @@
         end
       end
 
-      context "that needs an auth sanitizing" do
+      context "when it needs an auth sanitizing" do
```
Comment (abdulapopoola) [inline npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npmrc_builder_spec.rb:51]:
```suggestion
      context "when it needs auth sanitizing" do
```

File: npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npmrc_builder_spec.rb [hunk 5]
```diff
@@ -39,7 +39,7 @@
           .to eq(fixture("projects", "generic", "npmrc_auth_token", ".npmrc"))
       end
 
-      context "that needs an authToken sanitizing" do
+      context "when it needs an authToken sanitizing" do
```
Comment (abdulapopoola) [inline npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/npmrc_builder_spec.rb:42]:
```suggestion
      context "when it needs to sanitize the authToken" do
```

**Pull request merged**